syncopy/datatype/discrete_data.py

```diff
@@ -184,7 +185,12 @@ def trialtime(self):
 
     # Helper function that grabs a single trial
     def _get_trial(self, trialno):
-        return self._data[self.trialid == trialno, :]
+        this_trl = self.trialid == trialno
+        if not np.any(this_trl):
+            return self._data[None, :]
```

Comment:
Isn't this the same as 
```python
return self._data[()]
```
and why would we want to return the whole dataset in case the `trialno` does no exist?

Reply:
Ah sorry I want to return an empty slice e.g. self._data[0:0, :]